src/Common/src/Interop/Unix/libcurl/Interop.libcurl_easy.cs

```diff
@@ -77,5 +77,8 @@ internal static partial class libcurl
 
         [DllImport(Interop.Libraries.LibCurl)]
         public static extern int curl_easy_pause(SafeCurlHandle easy, int bitmask);
+
+        [DllImport(Interop.Libraries.LibSsl)]
+        internal static extern void SSL_CTX_set_client_cert_cb(IntPtr sslCtx, [MarshalAs(UnmanagedType.FunctionPtr)]client_cert_cb callback);
```

Comment:
This attribute shouldn't be necessary; it's the default, and we don't explicitly annotate it other places we marshal delegates.